web/src/CityContentSwitcher.tsx

```diff
@@ -44,18 +44,23 @@ const DisclaimerPage = lazyWithRetry(() => import('./routes/DisclaimerPage'))
 
 type PropsType = {
   cities: CityModel[]
-} & RouteComponentProps<{ cityCode: string; languageCode: string }>
+  languageCode: string
```

Comment:
isn't cityCode now missing?

Reply:
No, we just have to extract it manually using `useParams`.

Reply:
And why do we handle languageCode differently? Couldn't this also just be extracted using `useParams`?

Reply:
Because we already extracted it somewhere else and there is no need to extract it again. But yes, would be possible.